### ISSUE TITLE
Errors and warnings showing doubled

### ISSUE THREAD

**Issue by Zachu** [warpnet/SublimeLinter-salt-lint#16]
Console shows SublimeLinter running the lint as follows:
```sh
$ cat path/to/file/init.sls | /home/zachu/.local/bin/salt-lint --nocolour --severity /full/path/to/file/init.sls
```

This makes the salt-lint to run the linting for both to `stdin` file as well as the file from input. And this in turn shows the error messages doubled.

I tried to compare this linter to for example https://github.com/thomasmeeus/SublimeLinter-contrib-yamllint which doesn't seem to `cat` the source files into the linter and the code from https://github.com/SublimeLinter/SublimeLinter/blob/master/lint/linter.py and I believe yamllint works correctly because of `tempfile_suffix = 'yaml'` being provided and therefore making SublimeLinter first create tempfile for the code. So maybe this linter should also provide some `tempfile_suffix`?

Other possible fixes I could think of are removing the `${file}` from the `cmd` or setting `tempfile_suffix` to `-` as is stated in https://github.com/SublimeLinter/SublimeLinter/blob/master/lint/linter.py#L672.

**Comment by Zachu** [comment]
I guess the best choice would be to remove the `${file}` actually so that background linting without saving would work too.

**Comment by atezet** [comment]
The reason I added `${file}` was because when linting from `stdin`, `salt-lint` first writes everything to a temporary file before linting it. However, the tradeoff seems worth it when taking into account that background linting works now too. Also, I was under the assumption that adding the `${file}` argument would stop `stdin` linting, which it doesn't, causing the double warnings.

Thanks for the report!

**Comment by atezet** [comment]
Fixed in 3.0.0
